egs/vcc2020/vc1/local/recognize.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Copyright 2020 Nagoye University (Wen-Chin Huang)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+echo "$0 $*"  # Print the command line for logging
+. ./path.sh
+
```

Comment:
How about introducing `stage` like `run.sh`?
Then, we can also add `recog_stage` in `vc1/run.sh` and add that option when we call `local/recognize.sh`, e.g.,
```
local/recognize.sh --stage ${recog_stage}
```

Reply:
I can add this, but I wonder if this feature would be used, because the whole recognition stage is just one step in the complete VC flow, so users may hardly ever want to separately execute the recognition stages...?

Reply:
I see. I think we don't have to do it then.